config/locations/holding_locations.json

```diff
@@ -3206,7 +3206,6 @@
             "PH",
             "PM",
             "PG",
-            "PN",
```

Comment:
`lewis$nb`

Reply:
I don't know why this has every location checked off. It can go only to Engineering.